modules/local/modules/trimgalore/main.nf

```diff
@@ -88,8 +81,8 @@ process TRIMGALORE {
             ${meta.id}_2.fastq.gz
 
         cat <<-END_VERSIONS > versions.yml
-        ${getProcessName(task.process)}:
-            ${getSoftwareName(task.process)}: \$(echo \$(trim_galore --version 2>&1) | sed 's/^.*version //; s/Last.*\$//')
+        "${task.process}":
+            trim_galore: \$(echo \$(trim_galore --version 2>&1) | sed 's/^.*version //; s/Last.*\$//')
             cutadapt: \$(cutadapt --version)
         END_VERSIONS
 
```

Comment:
why not using the trimgalore nf-core module here ?

Reply:
Same answer again, my version is custom but not perhaps relevant for the wider community